tsconfig.json

```diff
@@ -5,14 +5,14 @@
     "baseUrl": "./",
     "outDir": "./dist/out-tsc",
     "forceConsistentCasingInFileNames": true,
+    "esModuleInterop": true,
```

Comment:
Please revert all changes in the tsconfig. Neither change is needed when i test

Reply:
All changes reverted.